src/app.js

```diff
@@ -1 +1,29 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFile() {
+  const pathes = process.argv.slice(2);
+
+  const pathExample = path.join(__dirname, pathes[0]);
+
+  const pathCopy = path.join(__dirname, pathes[1]);
```

Comment:
I think there is no need to join __dirname. Let users set absolute or relative path on their choice